starknet_py/net/models/address.py

```diff
@@ -26,29 +25,27 @@ def parse_address(value: AddressRepresentation) -> Address:
 
 
 def compute_address(
-    contract_hash: int, constructor_calldata: Sequence[int], salt: int
+    *,
+    class_hash: int,
+    constructor_calldata: Sequence[int],
+    salt: int,
+    deployer_address: int = 0,
```

Comment:
Do we need to introduce breaking changes here? We can just add an additional parameter.